doc/release/1.9.0-notes.rst

```diff
@@ -70,46 +139,213 @@ New features
+  documentation for `scipy.stats.combine_pvalues` has been expanded and improved.
+
+- Unlike other reduction functions, ``stats.mode`` didn't consume the axis
+  being operated on and failed for negative axis inputs. Both the bugs have been
+  fixed. Note that ``stats.mode`` will now consume the input axis and return an
+  ndarray with the ``axis`` dimension removed.
+
+- Replaced implementation of `scipy.stats.ncf` with the implementation from
+  Boost for improved reliability.
+
+- Add a `bits` parameter to `scipy.stats.qmc.Sobol`. It allows to use from 0
+  to 64 bits to compute the sequence. Default is ``None`` which corresponds to
+  30 for backward compatibility. Using a higher value allow to sample more
+  points. Note: it does not correspond to the output type.
+
+- Add a `integers` method to `scipy.stats.qmc.QMCEngine`. It allows sampling
+  integers using any QMC sampler.
+
+- improved the fit speed and accuracy of ``stats.pareto``
+
+- added ``qrvs`` method to ``NumericalInversePolynomial`` to match the
+  situation for ``NumericalInverseHermite``
+
+- faster random variate generation for ``gennorm`` and ``nakagami``
+
+- ``lloyd_centroidal_voronoi_tessellation`` has been added to allow improved
+  sample distributions via iterative application of Voronoi diagrams and
+  centering operations
+
+- Add `scipy.stats.qmc.PoissonDisk` to sample using the Poisson disk sampling
+  method. It guarantees that samples are separated from each other by a
+  given ``radius``.
+
+- Add `scipy.stats.pmean` to calculate the weighted power mean also called
+  generalized mean.
 
 
 *******************
 Deprecated features
 *******************
 
-`scipy.linalg` deprecations
-===========================
-
-
-`scipy.spatial` deprecations
-============================
-
-
+- Due to collision with the shape parameter ``n`` of several distributions,
+  use of the distribution ``moment`` method with keyword argument ``n`` is
+  deprecated. Keyword ``n`` is replaced with keyword ``order``. 
+- Similarly, use of the distribution ``interval`` method with keyword arguments
+  ``alpha`` is deprecated. Keyword ``alpha`` is replaced with keyword
+  ``confidence``.
+- The ``'simplex'``, ``'revised simplex'``, and ``'interior-point'`` methods
+  of `scipy.optimize.linprog` are deprecated. Methods ``highs``, ``highs-ds``,
+  or ``highs-ipm`` should be used in new code.
+- Support for non-numeric arrays has been deprecated from ``stats.mode``.
+  ``pandas.DataFrame.mode`` can be used instead.
+- The ``extradoc`` keyword of ``rv_continuous``, ``rv_discrete`` and
+  ``rv_sample`` now raises a DeprecationWarning, after having been deprecated in
+  documentation only for a long time already.
+
+********************
+Expired Deprecations
+********************
+There is an ongoing effort to follow through on long-standing deprecations.
+The following previously deprecated features are affected:
+- Object arrays in sparse matrices now raise an error
```

Comment:
You need a blank line before the beginning of the bulleted list.  (I was going to link to the comment about this in the "missing bits" document, but apparently the link to that document in the rendered docs was broken by some recent reshuffling of the docs files. I see if I can fix that and submit a PR.)

Reply:
Here's the link to the relevant section of the docs from a PR that fixes the reference to the "missing bits" file: [Bulleted lists](https://output.circle-artifacts.com/output/job/7265df6e-f1f4-4e37-9ca6-2cd6a7b903f1/artifacts/0/html-scipyorg/dev/contributor/missing-bits.html#bulleted-lists)